flink-runtime/src/test/java/org/apache/flink/runtime/operators/hash/MemoryHashTableTest.java

```diff
@@ -100,340 +101,371 @@
 	@Test
 	public void testDifferentProbers() {
 		final int NUM_MEM_PAGES = 32 * NUM_PAIRS / PAGE_SIZE;
-		
-		AbstractMutableHashTable<IntPair> table = new CompactingHashTable<IntPair>(serializer, comparator, getMemory(NUM_MEM_PAGES, PAGE_SIZE));
-		
+		testDifferentProbersCore(new CompactingHashTable<>(serializer, comparator, getMemory(NUM_MEM_PAGES, PAGE_SIZE)), NUM_MEM_PAGES);
```

Comment:
I think it is good practice to separate tests for different components into different test classes. That way you see directly for which component a test failed. I also keeps logic separated.